### ISSUE TITLE
Future development

### ISSUE THREAD

**Issue by eugeniobaglieri** [erdtman/spotify-ios-metadata#12]
Hi,
do you accept new pull requests or this repository should be considered depreacated and dead?

**Comment by erdtman** [comment]
Yes I would accept pull requests, but I´m not personally doing any active development.
